test/index.js

```diff
@@ -0,0 +1,15 @@
+import assert      from 'assert'
+import superagent  from 'superagent'
+import startServer from '../lib/server'
+
+startServer(8001)
+
+describe('/version', () => {
+  it('returns the current version', (done) => {
+    superagent.get('http://localhost:8000/version')
+              .end((err, res) => {
+                assert.deepEqual({ version: '0.0.1' }, JSON.parse(res.text))
+                done()
+              })
```

Comment:
Probably half-dozen one way, 6 the other, but checkout [`server.inject`](http://hapijs.com/api#serverinjectoptions-callback).